scripts/generate_protobuf_and_funcs.rb

```diff
@@ -441,6 +441,39 @@ def generate!
   // Named tokens in scan.l
   #{@scan_protobuf_tokens.join("\n  ")}
 }
+
+message SummaryResult {
+  message SummaryResultTable {
+    string table = 1;
+    // FIXME(duckinator): I AM NOT CONFIDENT Node IS RIGHT. THIS NEEDS TO BE VERIFIED.
+    Node context = 2;
```

Comment:
It is extremely unclear to me if `Node` is correct here, but it's the closest Protobuf thing I could find to the `Context` enum in Rust:

https://github.com/pganalyze/pg_query.rs/blob/23ab4ae1c989fe0dea919d4fee043993e97f6223/src/node_enum.rs#L5-L12

Is this right, or does there need to be a new thing added that's equivalent to `Context`?

Reply:
`Node` represents the full parse tree, so wouldn't work here. Instead you should add a protobuf enum that matches the `Context` enum in Rust.